ui/js/jquery.pods.js

```diff
@@ -210,6 +210,16 @@
                         }
                     } );
 
+                    // Check for unsaved open fields. (separate if's to prevent possible unneeded jQuery selector)
+                    if ( 'undefined' !== typeof postdata.method ) {
+	                    if ( 'save_pod' === postdata.method ) {
+	                        if ( $( 'tbody.pods-manage-list tr.pods-manage-row-expanded', $submittable ).length ) {
+		                        alert( PodsI18n.__( 'Not all field data is saved' ) );
```

Comment:
Some fields have changes that were not saved yet, please save them or cancel the changes before saving the Pod.